src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -231,28 +231,6 @@ class ResourceServiceSpec extends FlatSpec with Matchers with TestSupport with B
     runAndWait(service.deleteResource(resource, dummyUserInfo))
   }
 
-  it should "fail when user doesn't have alterpolicies permission" in {
```

Comment:
these 2 deleted tests are already (and more comprehensively) tested in ResourceRouteSpec